subworkflows/nf-core/gatk_joint_germline_variant_calling/main.nf

```diff
@@ -0,0 +1,117 @@
+//
+// Run GATK haplotypecaller for all input samples, merge them with genomicsdbimport, perform joint genotyping with genotypeGVCFS and recalibrate with variantrecalibrator & applyvqsr.
+//
+
+include { GATK4_HAPLOTYPECALLER     as HAPLOTYPECALLER     } from '../../../modules/gatk4/haplotypecaller/main'
+include { GATK4_GENOMICSDBIMPORT    as GENOMICSDBIMPORT    } from '../../../modules/gatk4/genomicsdbimport/main'
+include { GATK4_GENOTYPEGVCFS       as GENOTYPEGVCFS       } from '../../../modules/gatk4/genotypegvcfs/main'
+include { GATK4_VARIANTRECALIBRATOR as VARIANTRECALIBRATOR } from '../../../modules/gatk4/variantrecalibrator/main'
+include { GATK4_APPLYVQSR           as APPLYVQSR           } from '../../../modules/gatk4/applyvqsr/main'
+
+workflow GATK_JOINT_GERMLINE_VARIANT_CALLING {
+    take:
+    input                     // channel: [ val(meta), [ input ], [ input_index ], [] ]
+    run_haplotc               // channel: true/false run haplotypecaller portion of workflow or skip to genomicsdbimport when false
+    run_vqsr                  // channel: true/false run vqsr portion of subworkflow
+    fasta                     // channel: /path/to/reference/fasta
+    fai                       // channel: /path/to/reference/fasta/index
+    dict                      // channel: /path/to/reference/fasta/dictionary
+    sites                     // channel: /path/to/known/sites/file
+    sites_index               // channel: /path/to/known/sites/index
+    interval_file             // channel: /path/to/interval/file
+    joint_id                  // channel: joint id to replace individual sample ids with
+    allelespecific            // channel: true/false run allelespecific mode of vqsr modules
+    resources                 // channel: [[resource, vcfs, forvariantrecal], [resource, tbis, forvariantrecal], [resource, labels, forvariantrecal]]
+    annotation                // channel: [annotations, to, use, for, variantrecal, filtering]
+    mode                      // channel: which mode to run variantrecal: SNP/INDEL/BOTH
+    create_rscript            // channel: true/false whether to generate rscript plots in variantrecal
+    truthsensitivity          // channel: 0-100.0 truthsensitivity cutoff for applyvqsr
```

Comment:
I am a bit conflicted about passing so many value args to ApplyVQSr throughout the workflow. I would say that the new modules.config should contain the parameters.